packages/vitepress-plugin-git-changelog/src/client/composables/changelog.ts

```diff
@@ -29,9 +29,9 @@ export function useChangelog(pageData: Ref<PageData>) {
   })
 
   const authors = computed(() => {
-    const uniq = new Map<string, AuthorInfo>()
+    const uniq = new Map<string, AuthorInfo>();
 
-    commits.value.map(c => c.authors)
+    [...commits.value.map(c => c.authors), ...pageData.value.frontmatter.authors ?? []]
```

Comment:
Should we validate the props from `pageData.value.frontmatter.authors`?

Reply:
Yes, it needs to be validated, I pushed a fix to make sure it was `string[]`.